hnginternship5/ziki
# Fix : Update all links to homepage with logo

#### What does this PR do?
* It updates all links to homepage with the ziki logo
#### Task summary
* Remove the homepage text and replace it with a logo
* Update static logo with hompage link
#### How should this be manually tested?
* Visit any page that has a logo
* On logo click
* It should redirect you to the homepage
#### Screenshot[s]
![timeline-logo](https://user-images.githubusercontent.com/24655101/55839065-3d353b00-5b2f-11e9-8a6f-f0102c2f3f3a.png)

![blog-detail-logo](https://user-images.githubusercontent.com/24655101/55839068-40c8c200-5b2f-11e9-9c97-a80c1820459d.png)

![settings-logo](https://user-images.githubusercontent.com/24655101/55839091-59d17300-5b2f-11e9-8fce-c691ba664c34.png)
